sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCV2Suite.scala

```diff
@@ -2635,6 +2635,10 @@ class JDBCV2Suite extends QueryTest with SharedSparkSession with ExplainSuiteHel
     val tableIndex = indexes2.head
     assert(tableIndex.indexName() == "people_index")
 
+
+    val df = sql(s"SHOW INDEX ON TABLE h2.test.people")
+    checkAnswer(df, Seq(Row("people_index", "INDEX", Array("ID"), Map(), Map())))
```

Comment:
Is it possible to also test 

```
        tableIndex.columnProperties().asScala.toMap,
        tableIndex.properties().asScala.toMap))
```

Which are two empty maps here?

Reply:
The result about `SHOW INDEX`is :
```
+--------------+----------+----------+-----------------+----------+
|    Index Name|Index Type|   Columns|Column Properties|Properties|
+--------------+----------+----------+-----------------+----------+
|people_index_1|     INDEX|      [ID]|               {}|        {}|
+--------------+----------+----------+-----------------+----------+
```
Because in h2database, properties in `CREATE INDEX ` is not supported. So the `columnsProperties` and `properties` are empty.

Reply:
thanks for the clarification.